.github/workflows/rpm_amoy_profile_packager.yml

```diff
@@ -0,0 +1,323 @@
+name: rpm_amoy_profile
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
We need to change to master in bor as well.

Reply:
Updated